internal/driver/constant.go

```diff
@@ -1,12 +1,14 @@
 // -*- Mode: Go; indent-tabs-mode: t -*-
 //
-// Copyright (C) 2018-2019 IOTech Ltd
+// Copyright (C) 2018-2021 IOTech Ltd
 //
 // SPDX-License-Identifier: Apache-2.0
 
 package driver
 
-import sdkModel "github.com/edgexfoundry/device-sdk-go/pkg/models"
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2"
```

Comment:
The `v2` namespace will go away. So to reduce future changes for when that happens, might consider aliasing this to what we expect the namespace will be. Same in other files.

Reply:
I'm really not sure where this constants.go file would end up being, in v1 we put it in `clients` package but I doubt there will be import cycle in v2 if we do so. 

If we eventually still needs to update the import path (remove the `v2` namespace), I would prefer make the change till then.